### DISCUSSION TITLE
how cleariy open Github

### DISCUSSION THREAD

**Discussion by neimmusa** [community/community#146539] [New to GitHub]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

How to cleary open Github in short cut method

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @neimmusa ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ghostinhershell** [comment]
Hi @neimmusa! We hope you found what you were looking for. ✨

We're closing this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
